dhcpv6/dhcpv6.go

```diff
@@ -199,3 +201,84 @@ func EncapsulateRelay(d DHCPv6, mType MessageType, linkAddr, peerAddr net.IP) (D
 	outer.AddOption(&orm)
 	return &outer, nil
 }
+
+// IsNetboot function takes a DHCPv6 message and returns true if the machine
+// is trying to netboot. It checks if "boot file" is one of the requested
+// options, which is useful for SOLICIT/REQUEST packet types, it also checks
+// if the "boot file" option is included in the packet, which is useful for
+// ADVERTISE/REPLY packet.
+func IsNetboot(msg DHCPv6) bool {
```

Comment:
similarly to the above, what about making `IsNetboot` a method of `DHCPv6` ?

Reply:
I am not sure. Should we add it to the `DHCPv6` interface? As a method of `DHCPv6Message`? Or leave it as a function.